content/chainguard/chainguard-images/how-to-use/verifying-chainguard-images-and-metadata-signatures-with-cosign/index.md

```diff
@@ -21,20 +21,57 @@ toc: true
 
 All Chainguard Containers contain verifiable signatures and build-time SBOMs (software bills of materials), features that enable users to confirm the origin of each image built and have a detailed list of everything that is packed within.
 
-The following commands requires [Cosign](/open-source/sigstore/cosign/how-to-install-cosign/) and [jq](https://stedolan.github.io/jq/) to be installed on your machine in order to download and verify image attestations. It will pull detailed information about all signatures found for the provided image.
+This guide outlines how you can use Cosign to verify container image signatures and attestations, as well as download container attestations. 
```

Comment:
I wonder if download should be earlier, it seems more foundational. Maybe just "use Cosign to download and verify container image signatures and attestations."